### ISSUE TITLE
Range is valid multiple times per day

### ISSUE THREAD

**Issue by b3nj1** [biddster/node-red-contrib-time-range-switch#16]
I'm seeing time windows trigger in twice within a 24 hour period instead of the expected one time. In my case, I'm using sunrise to 10PM. I see the time window between sunrise and 10PM **plus** another window at 1AM (well before sunrise at my lat/long).

**Comment by b3nj1** [comment]
PR for this bug: #15

**Comment by electricJP** [comment]
I have the same issue, more or less, and it took me some time to understand: 
- I have a time range from sunset to 11 PM (for a PIR to control lights)
- a few times, lights went on during the night (say 1 AM)
I finally detected that "end time" will jump to the next day but not "start time". For example last night I was up late and I saw at 1:45 AM that the time range for my automation was "start time" = 2018-07-07 but "end time" was already 2018-07-08. A few hours later "start time" had updated to the right day.
Not sure why or what. Running on a PI with Home-Assistant.
